lib/MediaWords/Util/Bitly.pm

```diff
@@ -154,7 +154,7 @@ sub bitly_processing_is_enabled()
     my $config = MediaWords::Util::Config::get_config();
     my $bitly_enabled = $config->{ bitly }->{ enabled } // '';
 
-    return $bitly_enabled + 0;
+    return $bitly_enabled eq 'yes';
```

Comment:
YAML values are now booleans so they are not expected to return `yes` / `no` anymore but instead are set to a scalar value that evaluates to Perl's true or false. Please update your development's `mediawords.yml` and change all `['"]?(yes|no|true|false)['"]?` to `true` or `false` (without quotes). I've updated production's `mediawords.yml` manually so this (revert of) change would break on production.

Reply:
fixed